### PR TITLE
CDEC-513: Implement license verification system

### PR DESCRIPTION
**Pull request by tolusha**

### CODE REVIEW

File: installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java [hunk 1]
```diff
@@ -157,4 +159,10 @@ private Path getStorageFile() throws IOException {
         return storageFile;
     }
 
+    private void migrate(@Named("installation-manager.storage_dir_old") String storageDirOld) throws IOException {
```
Comment (sleshchenko) [inline installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java]: Are you assured that we need @Named here?

File: installation-manager-core/src/test/java/com/codenvy/im/facade/TestDownloadInstallationManagerFacade.java [hunk 2]
```diff
@@ -80,6 +81,8 @@
     private InstallManager             installManager;
     @Mock
     private DownloadManager            downloadManager;
+    @Mock
+    private LicenseManager             licenseManager;
```
Comment (sleshchenko) [inline installation-manager-core/src/test/java/com/codenvy/im/facade/TestDownloadInstallationManagerFacade.java]: Why do you add license manager mock and don't use it?

File: installation-manager-server-war/src/main/java/com/codenvy/im/service/LicenseService.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+/**
+ * @author Anatoliy Bazko
+ */
+@Path("/license")
+@RolesAllowed({"system/admin"})
+@Api(value = "license", description = "License manager")
+public class LicenseService {
+    private static final Logger LOG = LoggerFactory.getLogger(LicenseService.class);
+
+    protected final InstallationManagerFacade delegate;
```
Comment (sleshchenko) [inline installation-manager-server-war/src/main/java/com/codenvy/im/service/LicenseService.java]: Why do you use protected visibility here?

File: installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java (hunk 1 above)
Comment (tolusha) [inline installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java]: It is not actual anymore. I got rid of it.

File: installation-manager-core/src/test/java/com/codenvy/im/facade/TestDownloadInstallationManagerFacade.java (hunk 2 above)
Comment (tolusha) [inline installation-manager-core/src/test/java/com/codenvy/im/facade/TestDownloadInstallationManagerFacade.java]: Fixed

File: installation-manager-server-war/src/main/java/com/codenvy/im/service/LicenseService.java (hunk 3 above)
Comment (tolusha) [inline installation-manager-server-war/src/main/java/com/codenvy/im/service/LicenseService.java]: No need. I will fix.

File: installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java (hunk 1 above)
Comment (tolusha) [inline installation-manager-core/src/main/java/com/codenvy/im/managers/StorageManager.java]: It was copy/paste error

**Pull request merged**